detox/test/types/detox-global-tests.ts

```diff
@@ -28,6 +28,7 @@ describe("Test", () => {
 
     beforeAll(async () => {
         await device.reloadReactNative();
+        await device.resetAppState();
```

Comment:
why here? @noomorph / @jonathanmos

Reply:
@asafkorem  it is not a test per se, it is a typing/compiler error check thing.